docs/guides/postinstall.md

```diff
@@ -83,6 +83,16 @@ The steps to be followed vary depending upon the initramfs module loading mechan
          ```
 
     3. Run `sudo mkinitcpio -P`.
+ 
+- On EndeavourOS, which uses `dracut`
+
+    1. Create and edit a file at `/etc/dracut.conf.d/myflags.conf`, using e.g. `sudo vim /etc/dracut.conf.d/myflags.conf`
```

Comment:
Create and edit a file? Isn't the file available by default?

If not available, just do `echo "force_drivers+=\" apple-bce \"" | sudo tee /etc/dracut.conf.d/myflags.conf`

Reply:
No, you do have to create the file yourself for some reason.

Reply:
I think the file should be named "t2linux-modules.conf" for clarity. Otherwise LGTM